R/components.R

```diff
@@ -48,6 +48,17 @@ input <- function(name, defaultValue) {
   }
 }
 
+inputWithoutDefault <- function(name) {
+  function(inputId, ...) {
+    shiny.react::reactElement(
+      module = "@/shiny.blueprint",
+      name = name,
+      props = shiny.react::asProps(inputId = inputId, ...),
+      deps = blueprintDependency()
+    )
+  }
+}
+
```

Comment:
This code is redundant with `button`

I suggest either to reuse button() or this one for button _(which naming-wise it might make more sense)_

Reply:
`button` removed since `inputWithoutDefault` is more genral.